pombola/kenya/templates/core/person_bills.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'core/person_base.html' %}
+{% load url from future %}
+
+{% block title %}{{ object.name }} Bills Sponsored{% endblock %}
+
+{% block subcontent %}
+
+  {% if settings.ENABLED_FEATURES.bills %}
+    {% with object.bills_sponsored.count as bills_count %}
+      {% if bills_count %}
+        <h2>Bills Sponsored</h2>
+
+        <p>{{ object.name }} has sponsored {{ bills_count }} bills.</p>
```

Comment:
A very small thing, but this could be `bill{{ bills_count|pluralize }}` so we don't get "has sponsored 1 bills".